src/modules/keyboardmanager/README.md

```diff
@@ -0,0 +1,96 @@
+a particular shortcut, click the \<Remap Shortcut\> button to
+launch the Edit Shortcuts UI. When first launched, you are met
+with no predefined mappings and must click the \<+\> button to add
+a new remap. The Original shortcut is the shortcut you want to
+change and the New Shortcut is the shortcut you want to change it
+to. Ex. If you want Ctrl+C to paste, Ctrl+C is the Original
+Shortcut and Ctrl+V is the New Shortcut. There are a few rules to shortcuts:
+   
+- Shortcuts must begin with a modifier key (Ctrl, Shift, Alt, and Winkey)
+- Shortcuts must end with an action key (all non-modifier keys and non Fn keys)\* 
+- Shortcuts cannot be longer than 3 keys  
+
+![alt text][remapshort]
+
+### System Reserved shortcuts 
+- Winkey + L (Locking your computer) and Ctrl + Alt + Del cannot be remapped (as Original Shortcut) as they are reserved by the Windows OS.
+
+## Selecting the keys: Drop down + Type Key feature  
+To select a key in the remap or shortcut UI, you can use either the
+\<Type Key\> button or the drop downs. For the drop downs you select a key from the list and other drop downs will appear o Once you click the \<type key\> button a dialogue will pop up. From here, type the key/shortcut using your keyboard. Once you’re satisfied with the output, hold enter to continue. If you’d like to leave the dialogue, hold the ESC button.
+
+![alt text][dropdowntypekey]
+
+## Frequently Asked Questions
+
+- **Question**: *Can I use Keyboard Manager at my log-in screen?*
+  - No, Keyboard Manager is only available when PowerToys is running and doesn’t work on any password screen, including Run As Admin.
+
+- **Question**: *Do I have to turn off my computer for the remapping to take effect?*
+  - No, as of now, all you need to do is press apply.
+
+- **Question**: *Can I remap a shortcut to a single key?*
+  - No, we hope to support this soon.
+
+- **Question**: Why can’t I remap the Fn key?
+  - Fn keys are often hardware-based and are not available to the OS, meaning we aren’t able to intercept and remap the key
+
+- **Question**: *Where is the Mac/Linux profiles?*
+  - This is the beta release; we will have these features in our P1
+
+- **Question**: *Why can’t I remap a shortcut for a specific app?*
+  - This is the beta release; we will have these features in our P1
+
+- **Question**: *Will this work on video games?*
+  - It depends on how the game accesses your keys. Many games use low-level* keyboard hooks and as a result any remappings would not work.
+
+## Trouble shooting common problems:
+
+  - *My remapping’s are not working on a specific app / window*
+    
+      - *Could be one of two issues:*
+        
+          - Remappings will not work on an app / window if that window
+            is running as an admin (elevated) and PowerToys itself is
+            not running as admin. Try running PowerToys as an
+            administrator.
+        
+          - KBR intercepts keyboard hooks to remap your keys. Some apps
```

Comment:
Its KBM and Keyboard Manager in the rest of the doc, so we should do that here as well